README.md

```diff
@@ -0,0 +1,60 @@
+# Linux Mint Xed
+
+xed is a small and lightweight text editor.
+
+xed supports most standard editing features, plus several not found in your
+average text editor (plugins being the most notable of these).
+
+Although new features are always under development, currently xed has:
+
+    * Complete support for UTF-8 text
+    * Syntax highlighting
+    * Support for editing remote files
+    * Search and Replace
+    * Printing and Print Previewing Support
+    * File Revert
+    * A complete preferences interface
+    * Configurable Plugin system, with optional python support
+
+Some of the plugins, packaged and installed with xed include, among others:
+
+    * Word count
+    * Spell checker
+    * Change case of selected text
+    * File Browser
+    * Sort
+    * Insert Date/Time
+    * Tag list
```

Comment:
It might be nice to put all of the plugins here since you're updating it anyway.

Reply:
Double check me, but I'm pretty sure I got them all add in now. Also alphabetized the list.